package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "todo list webapp",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode=development"
```

Comment:
##  \[OPTIONAL\] Stretch Requirement.
- [ ] Would you mind adding a start script here from the `webpack-dev-server` for the smooth running of the application locally?